bundles/org.openhab.binding.velux/src/main/java/org/openhab/binding/velux/internal/handler/VeluxBridgeHandler.java

```diff
@@ -250,10 +260,7 @@ public void initialize() {
             logger.warn("initialize(): scheduler is shutdown, aborting the initialization of this bridge.");
             return;
         }
-        if (handleScheduler.isShutdown()) {
-            logger.trace("initialize(): handleScheduler is shutdown, aborting the initialization of this bridge.");
-            return;
-        }
+        handleScheduler = Executors.newSingleThreadExecutor();
```

Comment:
Please create the executor with a NamedThreadFactory so that you can give the threads names that you can tie to this binding/handler. Also make sure that the created threads are daemon.

Reply:
Ok.

Reply:
> Also make sure that the created threads are daemon.

@cpmeister see my issue with @fwolter further above; why do you want it to be daemon? It seems to reduce the performance to an unacceptable level.